package.json

```diff
@@ -34,13 +35,7 @@
   },
   "homepage": "https://github.com/thedumbterminal/jsonschema-bigquery#readme",
   "devDependencies": {
-    "eslint": "^5.6.1",
-    "eslint-config-standard": "^12.0.0",
-    "eslint-plugin-import": "^2.14.0",
-    "eslint-plugin-mocha": "^5.2.0",
-    "eslint-plugin-node": "^7.0.1",
-    "eslint-plugin-promise": "^4.0.1",
-    "eslint-plugin-standard": "^4.0.0",
```

Comment:
Are these plugins definitely not needed?

Reply:
we can add these back when we expand the eslint config i think some of them are now outdated and are part of eslint.